test/e2e_node/oomkiller_linux_test.go

```diff
@@ -249,5 +251,12 @@ func getOOMTargetContainerWithoutLimit(name string) v1.Container {
 			// use the dd tool to attempt to allocate huge block of memory which exceeds the node allocatable
 			"sleep 5 && dd if=/dev/zero of=/dev/null iflag=fullblock count=10 bs=10G",
 		},
+		SecurityContext: &v1.SecurityContext{
+			SeccompProfile: &v1.SeccompProfile{
+				Type: v1.SeccompProfileTypeRuntimeDefault,
+			},
+			AllowPrivilegeEscalation: &pfalse,
```

Comment:
it's better to use `ptr` library.

```suggestion
			AllowPrivilegeEscalation: ptr.To(false),
```

Reply:
Thanks, done